src/inngest/functions.ts

```
@@ -0,0 +1,322 @@
+      const transcript = await step.run("parse-transcript", async () => {
+        return JSONL.parse<StreamTranscriptItem>(transcriptText);
+      });
+
+      // Add speaker information
+      const transcriptWithSpeakers = await step.run("add-speakers", async () => {
+        // Log transcript data for debugging
+        console.log(`Transcript contains ${transcript.length} total items`);
+
+        // Filter out invalid transcript items first
+        const validTranscript = transcript.filter((item: StreamTranscriptItem) => {
+          const isValid = item && item.user_id && item.text && item.created_at;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)